specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/ServerDevOpsAudit.json

```diff
@@ -16,13 +16,13 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditPolicyName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditingSettingsName}": {
       "get": {
         "tags": [
-          "DevOpsAudit"
+          "ServerDevOpsAudit"
         ],
-        "description": "Gets a server's DevOps audit policy.",
-        "operationId": "ServerDevOpsAuditPolicies_Get",
+        "description": "Gets a server's DevOps audit settings.",
+        "operationId": "ServerDevOpsAuditSettings_Get",
```

Comment:
Must this operation id must change? as change operation id is a breaking change. 
Noticed you also change this file name, can we know the reason?  this will lead to  tool can not detect breaking change.

Reply:
This is a new swagger, its was check-in last week with a large check-in (not intentionally).
so there no SDKs published yet, so there is no breaking change here.